docs/man_pages/publishing/appmanager-groups.md

```diff
@@ -17,8 +17,8 @@ Command | Description
 [appmanager upload android](appmanager-upload-android.html) | Builds the project for Android and uploads the application to Telerik AppManager.
 [appmanager upload ios](appmanager-upload-ios.html) | Builds the project for iOS and uploads the application to Telerik AppManager.
 [appmanager upload wp8](appmanager-upload-wp8.html) | Builds the project for Windows Phone and uploads the application to Telerik AppManager.
-[appmanager livesync](appmanager-livesync.html) | Publish a new update of your application in Telerik AppManager.
-[appstore](appstore.html) | Lets you work with your iTunes Connect account.
+[appmanager livesync](appmanager-livesync.html) | Publishes a new Telerik AppManager LiveSync update of your published application.
+[appstore](appstore.html) | Allows interaction with iTunes Connect.
```

Comment:
Lets you work with your iTunes Connect account.

Apply to all `[appstore](appstore.html) |` instances.